### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by mysunshine92**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
 > 
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE

```

### CODE REVIEW

**Comment by yagonobre** [comment]
Thanks  for the fix @mysunshine92, can you squash the commits?
/ok-to-test

**Comment by neolit123** [comment]
thanks for the PR.
why does it include all these godep changes?

**Comment by mysunshine92** [comment]
@yagonobre thank you, I have squashed the commits

File: cmd/kubeadm/app/util/etcd/etcd.go [hunk 1]
```diff
@@ -356,7 +356,7 @@ func (c Client) WaitForClusterAvailable(delay time.Duration, retries int, retryI
 	return false, errors.New("timeout waiting for etcd cluster to be available")
 }
 
-// CheckConfigurationIsHA returns true if the given InitConfiguration etcd block appears to be an HA configuration.
+// CheckConfigurationIsHA returns true if the given init configuration etcd block appears to be an HA configuration.
```
Comment (yliaog) [inline cmd/kubeadm/app/util/etcd/etcd.go]: InitConfiguration is a struct, better to keep as is

Comment (yagonobre) [reply cmd/kubeadm/app/util/etcd/etcd.go]: +1

Comment (yagonobre) [reply cmd/kubeadm/app/util/etcd/etcd.go]: same to `EtcdCluster`

**Review by yagonobre — APPROVED** [review]
LGTM, thanks @mysunshine92

**Comment by neolit123** [comment]
/approve
for cmd/kubeadm
("pod" should be upper case always, but...)

**Comment by mysunshine92** [comment]
cc @yagonobre

**Comment by mysunshine92** [comment]
cc @yagonobre would you like to add approved, lgtm labels,thank you.

**Comment by yagonobre** [comment]
Hi @mysunshine92, you need aprove from sig/api-machinery.
/lgtm

**Comment by mysunshine92** [comment]
@smarterclayton can you help to add approved labels,thank you

**Comment by mysunshine92** [comment]
@smarterclayton can you help to add approved labels,thank you

**Comment by mysunshine92** [comment]
cc @yagonobre would you like to add approved, lgtm labels,thank you.

**Pull request closed without merging**